libs/sdk-core/src/swap_in/swap.rs

```diff
@@ -133,6 +133,9 @@ impl BTCReceiveSwap {
     }
 
     /// Create a [SwapInfo] that represents the details of an on-going swap.
+    ///
```

Comment:
This comment seems redundant now.

Reply:
Done, removed it: 861d688edd62745e45be90b0106dd2d481a8b988